pkg/operator/README.md

```diff
@@ -70,6 +70,61 @@ make generate
 go run ./cmd/aro operator master
 ```
 
+### How to create & publish ARO Operator image to ACR/Quay
+
+1. Login to AZ
+  ```bash
+  az login
+  ```
+
+2. Install Docker according to the steps outlined in [Prepare Your Dev Environment](../../docs/prepare-your-dev-environment.md)
+
+3. Publish Image to ACR
+
+   * Pre-requisite:
+     ```
+     ACR created in Azure Portal with Name ${USER}aro
+     2GB+ of Free RAM
+     ```
+
+    * Setup mirroring environment variables
+      ```bash
+      export DST_ACR_NAME=${USER}aro
+      export SRC_AUTH_QUAY=$(echo $USER_PULL_SECRET | jq -r '.auths."quay.io".auth')
+      export SRC_AUTH_REDHAT=$(echo $USER_PULL_SECRET | jq -r '.auths."registry.redhat.io".auth')
+      export DST_AUTH=$(echo -n '00000000-0000-0000-0000-000000000000:'$(az acr login -n ${DST_ACR_NAME} --expose-token | jq -r .accessToken) | base64 -w0)
+      ```
+
+    * Login to the Azure Container Registry
+      ```bash
+      docker login -u 00000000-0000-0000-0000-000000000000 -p "$(echo $DST_AUTH | base64 -d | cut -d':' -f2)" "${DST_ACR_NAME}.azurecr.io"
+      ```
+
+4. Publish Image to Quay
+
+    * Pre-requisite:
+      ```
+      Quay account with repository created
+      2GB+ of Free RAM
+      ```
+
+    * Setup mirroring environment variables
+      ```bash
+      export DST_QUAY=<quay-user-name>/<repository-name>
+      export SRC_AUTH_QUAY=$(echo $USER_PULL_SECRET | jq -r '.auths."quay.io".auth')
+      export SRC_AUTH_REDHAT=$(echo $USER_PULL_SECRET | jq -r '.auths."registry.redhat.io".auth')
```

Comment:
This is used for mirroring.  It's not actually needed for this guide here.  I would suggest removing this section.